numpy/core/src/multiarray/numpyos.c

```diff
@@ -15,7 +15,11 @@
 
 #ifdef HAVE_STRTOLD_L
 #include <stdlib.h>
-#include <xlocale.h>
+#ifdef HAVE_XLOCALE_H
+    #include <xlocale.h>
+#else
+    #include <locale.h>
```

Comment:
locale.h is already included so this can be removed
please also add a comment, e.g. `xlocale.h is included in locale.h on some systems`